.travis.yml

```diff
@@ -22,7 +22,11 @@ before_install:
   - if [ "$TRAVIS_SECURE_ENV_VARS" != "false" ]; then
         openssl aes-256-cbc -K $encrypted_1ef8dfbdb114_key -iv $encrypted_1ef8dfbdb114_iv -in travis.tar.gz.enc -out travis.tar.gz -d;
         tar -xzf travis.tar.gz;
-        export INTEGRATION_TEST_FLAGS="-Dit.spanner=true -Dit.storage=true -Dit.config=true -Dit.pubsub=true -Dit.logging=true";
+        export INTEGRATION_TEST_FLAGS="-Dit.spanner=true -Dit.storage=true -Dit.config=true -Dit.pubsub=true -Dit.logging=true
+            -Dit.jpa-sample=true
```

Comment:
Can we just use `-Dit.cloudsql`? This would indicate that the intent is to run all Cloud SQL tests, and it would be consistent with the other `it.*` flags.